src/app/projects/project-list/project-list-item.component.html

```diff
@@ -1,13 +1,15 @@
-<td class="col-name">{{project.name}}</td>
-<td class="col-group">{{project.groupName}}</td>
-<td class="col-provider">{{providerHostname}}</td>
-<td class="col-status"><span class="not-available">N/A</span></td>
+<td class="col-name url-link" [routerLink]="['/project', project.projectId]"><a [routerLink]="['/project', project.projectId]">{{project.name}}</a></td>
+<td class="col-group" [routerLink]="['/project', project.projectId]">{{project.groupName}}</td>
+<td class="col-provider" [routerLink]="['/project', project.projectId]">{{providerHostname}}</td>
+<td class="col-status" [routerLink]="['/project', project.projectId]"><span class="not-available">N/A</span></td>
```

Comment:
I think it's best to skip the `routerLink` on the `<td>` elements and only relying on the `<a>` tag inside the first cell.

You should also remove the `cursor` CSS from `src/scss/project-list-item.component.scss` then

Reply:
I tested your branch, and <kbd>Ctrl+Click</kbd> on the project name both redirect the current tab and opens a new tab with the project, which is an artifact of the parent also having an `onclick` override via this `routerLink` directive

Reply:
Cool, i missed the <kbd>Ctrl+Click</kbd> behaviour. 

Leaving it as only the <a> with the routerlink now.